### ISSUE TITLE
Sandstorm Wekan Grain 'Members' addition failing to show members

### ISSUE THREAD

**Issue by ertanalytics** [wekan/wekan#723]
Cannot see invitees as members when you are the primary member.  After fresh install of sandstorm on a personal https:// URL, I invited a mailinator address as a test.  Invite sent perfect, was able to view the board and created a user account on sandstorm however the new user never appears in 'Members' to be able to assign them to cards.  Furthermore the user addition does not even appear in the 'Activities' listing, it acts like it never happened.

I tried the inverse by creating a board with the new user account and inviting the original user and that appears to work.  

Tried reinstalling the grain, restoring from backup, but it is the same issue.  Seems like the member list will not be altered from this original backup so the worst work around is retyping all the cards and activities.  Please supply some ideas to restore the existing board while allowing the addition of new member assignment.

**Comment by dwrensha** [comment]
1. Did you share access with "can edit" permissions?
  2. Does the new user have a different "preferred handle" from the first user? (Currently necessary due to a bug in the Wekan package. Should be fixed soon.)

**Comment by ertanalytics** [comment]
I started with "can edit", but tried both and then change user types in sandstorm from admin to user to visitor.  They are jevans@mailinator.com and ddooley@mailinator.com, not sure if that is too similar with the bug or not.  When I created a board with the jevans username I was able to share and view both other members including the original primary admin account.  I am only using email authentication with codes for logins.  I am also on the latest version of everything stable if that helps.

**Comment by ertanalytics** [comment]
David - I was able to get the users to appear in the members list by doing this procedure:

1.  Invite user to sandstorm server
2.  Once the user has created an account, make sure they are set to 'User' status before sending the Wekan board invitation.
3.  As the user, login, create the account, then open and log into the board.

I will mark this as resolved for now since this seems to serve my purposes.  It might help to do some testing around this if it is intended to be smoother.  All my failed attempts are visible on those mailinator emails, I changed the usernames for privacy reasons on my actual site.
